### DISCUSSION TITLE
usage with jest

### DISCUSSION THREAD

**Discussion by bboydflo** [dmtrKovalenko/odiff#14] [Q&A]
Hi, I am using jest together with `https://github.com/americanexpress/jest-image-snapshot` for visual regression, but my snapshots are quite large and `jest-image-snapshot` is flaky. I was thinking to use odiff to do the image diff, are there any plans to support that?

or maybe you have some other ideas on how odiff could be integrated with jest?

**Comment by dmtrKovalenko** [comment]
That would be a nice addition, I will make a PR in jest image snapshot to allow odiff as optional instrument

**Reply by tujoworker** [reply]
I agree. Would be really nice. Did you ever go further with this?

**Reply by dmtrKovalenko** [reply]
I forgot 😞
